lib/services/internalEngine/index.js

```diff
@@ -546,14 +491,15 @@ class InternalEngine {
    * @returns {*}
    */
   deleteIndex () {
+    // TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)